File: devlibrary/src/main/java/com/example/devlibrary/network/SchedulerUtils.java
package com.example.devlibrary.network;

/**
 * Created by Cy on 2018/9/27.
 */
public class SchedulerUtils {

    public static <T> IoMainScheduler<T>  ioToMain() {
        return new IoMainScheduler<T>();
    }

}

File: baselib/src/main/java/com/wan/baselib/network/SchedulerUtils.kt
package com.wan.baselib.network

/**
 * Created by Cy on 2018/9/27.
 */
object SchedulerUtils {
    @JvmStatic
    fun <T : Any> ioToMain(): IoMainScheduler<T> {
        return IoMainScheduler()
    }
}